src/api.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+import JSONbig from 'json-big'
+
+// Defines the API instance in Axios with special handling for big integers.
+export const apiInstance = axios.create({
```

Comment:
Can we rename the instance to `axiosInstance`?  if we rename this file to `axios.ts`, will that cause any import conflicts with the main `axios` package.

Reply:
Done

Reply:
And yes, as long as the import is prefixed by `@/` it's a relative import and it can have the same names as the axios package.